File: 프로그래머스/unrated/181932. 코드 처리하기/코드 처리하기.py
def solution(code):
    answer = ''
    mode = 0
    for i in range(len(code)):
        if code[i] == "1":
            mode = 1 - mode
        elif i % 2 == mode:
            answer += code[i]
    return "EMPTY" if answer == "" else answer

File: 프로그래머스/0/181932. 코드 처리하기/코드 처리하기.cs
using System;

public class Solution {
    public string solution(string code) {
        int mode = 0;
        string ret = "";
        for (int i=0; i<code.Length; i++)
        {
            if (code[i].ToString() == "1")
            {
                mode = 1-mode;
                continue;
            }
            
            if (i%2==mode)
            {
                ret += code[i];
            }           
        }
        if (ret=="") ret="EMPTY";
        return ret;
    }
}